Scripts/git/pre-commit-validator.rb

```diff
@@ -0,0 +1,88 @@
+require_relative '../sdk_utilities'
+require_relative '../sdk_constants'
+require 'fileutils'
+
+class FileComparator
+  def self.compare(file1_path, file2_path)
+    unless File.exist?(file1_path) && File.exist?(file2_path)
+      return false
+    end
+
+    file1 = File.open(file1_path)
+    file2 = File.open(file2_path)
+
+    file1.each_line.zip(file2.each_line) do |line1, line2|
```

Comment:
you can use `FileUtils.identical?(file1, file2)` instead

Reply:
I wouldn't completely remove this method though, you might want to be able to throw an error if the file isn't found but once you have a file handler you can just use the mentioned method

Reply:
Updated 🙏